backend/hitas/calculations/improvement.py

```diff
@@ -164,7 +164,12 @@ def calculate_improvement(
 ) -> ImprovementCalculationResult:
     deprecation_result = None
 
-    if index_check and (index_check.completion_date_index is None or index_check.calculation_date_index is None):
+    if index_check and (
+        index_check.completion_date_index is None
+        or index_check.completion_date_index == 0
```

Comment:
as we now have this https://github.com/City-of-Helsinki/hitas/blob/master/backend/hitas/oracle_migration/runner.py#L116-L118 it would be better to enforce index value must be greater than 0 instead of greater than or equal to zero. Allowing setting the index value to zero does not make sense anymore :thinking:

Reply:
these checks can now be removed

Reply:
👍